src/test/java/com/wikia/webdriver/elements/oasis/templateclassificiation/EntryPoint.java

```diff
@@ -0,0 +1,55 @@
+package com.wikia.webdriver.elements.oasis.templateclassificiation;
```

Comment:
Why is oasis on the first level ? It should be functionality
we have to follow the current agreed architecture:
https://docs.google.com/document/d/1vroDsRxEHnI7mcU6i6-wvV1lQyCJBn2cKlzoFl1w6yk/edit

Elements
 Functionality
 generalizing.components (skin independent)
 generalizing.pages (skin independent)
 Oasis
 page
  page1
  page2
 component
  component 1 
  component 2
  component 3